INSTALLATION

```diff
@@ -2,13 +2,8 @@ These are pointers rather than complete notes on installation.
 
 # Requirements
 
-Use of the QGIS (2.x) plugin requires that sqlalchemy
-(<0.9) and geoalchemy python modules are installed.
-You can use `pip install -r src/NZGBplugin/requirements.txt`
```

Comment:
Should we update and retain QGIS Plugin install instructions

Reply:
Is this only for QGIS 2.x though or all version? Sorry I might have misunderstood.